app/services/solidus_bolt/base.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'httparty'
+
+module SolidusBolt
+  class Base
+    def initialize(*args); end
+
+    def call
+      raise NotImplementedError
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def preferred_api_base_url
+      api_base_urls = {
+        'development' => 'https://api-sandbox.bolt.com',
+        'test' => 'https://api-sandbox.bolt.com',
+        'staging' => 'https://api-staging.bolt.com',
+        'production' => 'https://api.bolt.com'
+      }
+      api_base_urls[Rails.env]
```

Comment:
I think we shouldn't make a mapping here. I have seen projects with a lot of environments and it would make it hard to customize.

I think we should let to the dev to set it from outside.

Reply:
This problem has been solved by using `SolidusBolt::BoltConfiguration` now